apps/emqx_bridge_http/src/emqx_bridge_http_connector.erl

```diff
@@ -507,8 +508,8 @@ resolve_pool_worker(#{pool_name := PoolName} = State, Key) ->
 on_get_channels(ResId) ->
     emqx_bridge_v2:get_channels_for_connector(ResId).
 
-on_get_status(_InstId, #{pool_name := PoolName, connect_timeout := Timeout} = State) ->
-    case do_get_status(PoolName, Timeout) of
+on_get_status(InstId, #{pool_name := InstId, connect_timeout := Timeout} = State) ->
```

Comment:
This creates an assert that `InstId` and `pool_name` are equal, and the function will crash with `function_clause` otherwise. Is this the intention?

Reply:
answer is yes :)